app/assets/javascripts/analytics/_pageViews.js

```diff
@@ -1,7 +1,75 @@
 (function (GOVUK) {
   GOVUK.GDM.analytics.pageViews = {
     'init': function () {
+      this.setCustomDimensions();
       GOVUK.analytics.trackPageview();
+    },
+
+    'setCustomDimensions': function() {
+      var search = GOVUK.GDM.analytics.location.search(),
+          filterGroupDimension = [],
+          numberOfOpportunites,
+          dimensions,
+          pairs,
+          pair,
+          i,
+          j;
+
+      // check that we're on the catalogue page for opportunites
+      if (GOVUK.GDM.analytics.location.pathname() === "/digital-outcomes-and-specialists/opportunities") {
```

Comment:
Sure thing.  Let's ask @tombye when he gets back about hardcoding urls in JS, [as it seems to be what we've gone with before](https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/0f3613a0f2cfdd67d78bc91b8e2ae3f456dd49c1/app/assets/javascripts/analytics/_virtualPageViews.js#L16).